hasura.planx.uk/metadata/databases/default/tables/public_magic_links.yaml

```diff
@@ -0,0 +1,37 @@
+table:
+  name: magic_links
```

Comment:
nit: should this specifically be `lps_magic_links` ? Not to be confused with pre-existing resume or payment magic links which because not token-based don't need to be stored in same way?

Reply:
Good shout, updated!